rubin_sim/movingObjects/directObs.py

```diff
@@ -76,7 +76,7 @@ class DirectObs(BaseObs):
         observations (in degrees).
         Default 10 degrees.
     """
-    def __init__(self, footprint='camera', rFov=2.45, xTol=5, yTol=3,
+    def __init__(self, footprint='camera', rFov=1.75, xTol=5, yTol=3,
```

Comment:
yes, agreed. To match MAF the circular fov is 1.75 deg, and the camera sets it own circle (because it is always needing the same circle size). So the 1.75 isn't relevant for the camera. 
(just commenting here for completeness).